front/app/components/PageForm/index.tsx

```diff
@@ -67,14 +67,23 @@ const PageForm = ({
   const page = usePage({ pageId });
   const appConfig = useAppConfiguration();
 
-  const schema = object({
-    nav_bar_item_title_multiloc: validateMultiloc(
-      formatMessage(messages.blankTitleError)
-    ),
+  const multilocFieldsValidation = {
     title_multiloc: validateMultiloc(formatMessage(messages.blankTitleError)),
     body_multiloc: validateMultiloc(
       formatMessage(messages.blankDescriptionError)
     ),
+  };
+
+  // If we're creating a new page, there is no pageId
+  // And also no slug field yet.
+  if (pageId) {
```

Comment:
Just for consistency and to not have several ways of implementing the same functionality, it would be better to add this condition directly in the schema below - in the same way as with `slug` and `hideSlugInput`.